README.md

```diff
@@ -48,7 +48,7 @@ X.test = matrix(0, 101, p)
 X.test[,1] = seq(-2, 2, length.out = 101)
 
 # Train a causal forest.
-W = rbinom(n, 1, 0.5)
+W = rbinom(n, 1, 0.25 + 0.5 * (X[,1] > 0))
```

Comment:
I think we should remove this update, as it's not related to the OOB prediction change.